src/sdk/models/entityTypes.ts

```diff
@@ -11,6 +11,44 @@ export interface EntityType extends IEntityIdentifier<string>, IAuditedEntity {
 	disabled?: boolean;
 }
 
+export interface BlockListEntityType extends EntityType {
+	kind: 'block_list';
```

Comment:
Did we want the discriminators on these? Seems to work well but the jobs examples (TransferJob, PolicyJob, etc.) don't seem to do this.